Competition2017/src/competition/subsystems/climbing/commands/IntakeAtLowPowerForTimeCommand.java

```diff
@@ -0,0 +1,61 @@
+package competition.subsystems.climbing.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.climbing.ClimbingSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class IntakeAtLowPowerForTimeCommand extends BaseClimbingCommand {
+    private DoubleProperty lowPowerProp;
+    private DoubleProperty timeoutProp;
+    private double lowPower;
+    private double timeout;
+    private double currentTime;
+
+    @Inject
+    public IntakeAtLowPowerForTimeCommand(ClimbingSubsystem climbingSubsystem, XPropertyManager propMan) {
+        super(climbingSubsystem);
+        lowPowerProp = propMan.createPersistentProperty("very low power for climbing intake command", 0.1);
+        timeoutProp = propMan.createPersistentProperty("the time in seconds for very low power intake", 0.5);
+    }
+
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+        if (timeoutProp != null) {
+            timeout = timeoutProp.get();
+        }
+        if (lowPowerProp != null){
+            lowPower = lowPowerProp.get();
+        }
+    }
+
+    @Override
+    public void execute() {
+        climbingSubsystem.ascendAtPower(lowPower);
+    }
+
+    public boolean isFinished() {
+        currentTime = Timer.getFPGATimestamp();
+        return currentTime >= timeout;
```

Comment:
since `Timer.getFPGATimestamp()` returns the number of seconds since the robot was started, this method will probably return TRUE all the time.